x-pack/legacy/plugins/siem/server/lib/timeline/types.ts

```diff
@@ -204,21 +208,36 @@ export const AllTimelineSavedObjectRuntimeType = runtimeTypes.type({
 export interface AllTimelineSavedObject
   extends runtimeTypes.TypeOf<typeof AllTimelineSavedObjectRuntimeType> {}
 
-export interface ExportTimelineRequestParams {
-  body: { ids: string[] };
-  query: {
-    file_name: string;
-    exclude_export_details: boolean;
-  };
+/**
+ * Import/export timelines
+ */
+
+export interface ExportTimelinesQuery {
+  file_name: string;
+  exclude_export_details: 'true' | 'false';
+}
+
+export interface ExportTimelinesRequestBody {
+  ids: string[];
 }
 
 export type ExportTimelineRequest = KibanaRequest<
```

Comment:
This is only used by `getExportTimelineByObjectIds`, which could be modified to take `ids` instead of the full request.